src/App.tsx

```diff
@@ -1,222 +1,146 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import React, {
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
 
-  useEffect(() => {
-    // focus the element with `ref={newTodoField}`
-    if (newTodoField.current) {
-      newTodoField.current.focus();
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [currentTodos, setCurrentTodos] = useState<Todo[]>([]);
+  const [isErrorShow, setIsErrorShow] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isRemoving, setIsRemoving] = useState(false);
+  const [currentTodoId, setCurrentTodoId] = useState<number[]>([]);
+
+  const onError = () => {
```

Comment:
```suggestion
  const handleError = () => {
```
Follow these rules on naming functions:
![image](https://user-images.githubusercontent.com/96409563/211518298-9964a68e-6ef5-40f4-9d3e-73f225cc7ce2.png)

Reply:
fixed